.gitignore

```diff
@@ -73,3 +73,5 @@ aws-android-sdk-testutils/src/main/res/raw/testconfiguration.json
 
 # AWSCoreRuntime libs
 aws-android-sdk-core/libs
+
+sample*
```

Comment:
Why do we need this change?

Reply:
I was wondering if we want to have a testing sample app module locally. Just making sure it does not get pushed in case anyone has it. I don't have any strong suggestions on this and can totally remove the ignore pattern.

Reply:
If you want you can this in your local ignore, dont need to commit to the repo.